integration/src/test/java/org/mule/test/integration/streaming/LogRepeatableStreamTestCase.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.test.integration.streaming;
+
+import static java.lang.String.format;
+import static org.apache.http.entity.ContentType.APPLICATION_JSON;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mule.runtime.http.api.HttpConstants.HttpStatus.OK;
+import org.mule.runtime.core.api.util.IOUtils;
+import org.mule.tck.junit4.rule.DynamicPort;
+import org.mule.test.AbstractIntegrationTestCase;
+
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class LogRepeatableStreamTestCase extends AbstractIntegrationTestCase {
```

Comment:
this doesn't really represent what you're testing. Logging a repeatable stream works fine. The problem is when you log a message which uses several DW expressions over the same stream. Either rename the class or use allure to hint that